todo.html

```diff
@@ -0,0 +1,335 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" rel="stylesheet"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>VanillaJS &#8226; TodoMVC</title>
+</head>
+<body style="background-color:rgb(234, 238, 238)">
+    <div class="grid-container">
+        <div class="header">
+            <h1>todos</h1>
+            <input calss= "addTask" type="text" id="addTask" placeholder="What needs to be done?"><br/><br/>
+            <div id="tasks">
+
+            </div><br/><br/>
+
+            <div class="flex-container">
+                <div id="numb"></div>
+                <div> 
+                <button type="button" class="btn active" id="all" onclick="AllTasks()">All</button>
+                <button type="button" class="btn" id="active" onclick="ActiveTasks()">Active</button>
+                <button type="button"class="btn" id="complited" onclick="ComplitedTasks()">Complited</button></div>
+            </div>
+        </div>
+    </div>
+
+    <script>
+        var activeTasks, complitedTasks, allTasks;
+        var numOfActiveTasks = 0;
+        var numbOfComplited = 0;
+        if(localStorage.getItem("tasks") == null || localStorage.getItem("tasks") == ""){
+            activeTasks = [];
+            complitedTasks = [];
+            allTasks = [];
+            localStorage.setItem("tasks", allTasks);
+            localStorage.setItem("active", activeTasks);
```

Comment:
Why are you storing both, only `tasks` should be enough. You can always filter those that are active